### ISSUE TITLE
Cobb County, GA Address Points Requested

### ISSUE THREAD

**Issue by justinelliotmeyers** [openaddresses/openaddresses#2497]
reached out to E911 to see if they have the gis data (https://cobbcounty.org/index.php?option=com_content&view=article&id=6988&Itemid=2813 )

**Comment by justinelliotmeyers** [comment]
Also sent e-mail to assessor's office: cobbtaxassessor@cobbcounty.org

**Comment by justinelliotmeyers** [comment]
emailed assesor, wrote to sheriff - also wrote on both facebook's wall and twitter accounts.  Full speed ahead!

**Comment by trescube** [comment]
Hit them up on tinder and eharmony, too!

**Comment by justinelliotmeyers** [comment]
well they don't know how to use reply button on e-mail... so that's not a bad idea... haha

**Comment by trescube** [comment]
Swipe right for open data

**Comment by justinelliotmeyers** [comment]
Good Morning,
 
I am forwarding your inquiry to a couple of GIS professionals within Cobb County Government. One of them should be able to assist with your request.
 
Regards,
 
Sean Kelly 
Deputy Chief Appraiser 
Cobb County Board of Tax Assessors 
(770) 528-3100

**Comment by trescube** [comment]
w00t!

**Comment by migurski** [comment]
Let's be kind, these threads are all public forever.

**Comment by justinelliotmeyers** [comment]
@migurski I always am regardless if we receive data or not. But sometimes I have to go a different route if e-mail and phone calls do not work ;)

**Comment by justinelliotmeyers** [comment]
You should be hearing from one of the two individuals I forwarded the request to. Either Seth Agyepong (GIS Supervisor in the Tax Assessors Office) or Jennifer Lana (Cobb County GIS Manager).
 
Your Welcome
 
Sean Kelly 
Deputy Chief Appraiser 
Cobb County Board of Tax Assessors 
(770) 528-3100

**Comment by justinelliotmeyers** [comment]
Good morning,
We do have a parcel dataset in GIS format and a tabular format that contains ownership information. 
Please visit the link below for data description and pricing information:

https://cobbcounty.org/index.php?option=com_content&view=article&id=5813&Itemid=2157

Thanks, 

Seth Agyepong
GIS Supervisor
Cobb County Tax Assessors Office
736 Whitlock Ave., Suite 200
Marietta, Ga. 30064
Tel. #: 770-528-3143 (Direct)
          770-528-3100 (General)
Fax #: 770-528-3118
 
www.cobbassessor.org

**Comment by justinelliotmeyers** [comment]
emailed back and asked if address points were situs, and also asked about terms of use if it is purchased.

**Comment by b-may** [comment]
I just talked to the GIS Manager for Cobb County to verify their pricing for parcels geometry - $22,000. Yep, that's the price. AND they assert copyright over the data as well. So, not only is the pricing outrageous, but beware of copyright issues in Georgia. The state statutes for public records are generally very liberal and promote free / cost open access, however, an exception for GIS data was added to the statutes to support "cost-recovery" (in 1999?). It allows agencies to charge whatever they want to support cost recovery and copyright the data. 

Interestingly enough, the CAMA data is $2. Kind of silly.

**Comment by nvkelso** [comment]
@b-may Thanks for the update! What is the CAMA data?

**Comment by b-may** [comment]
CAMA = Computer Assisted Mass Appraisal data, i.e. real property taxroll data like owner info, sales info, site address, valuations, land use, building info, etc.

**Comment by nvkelso** [comment]
This cost is prohibitive, but we think we have an alt source.

**Comment by justinelliotmeyers** [comment]
found this tonight in an interactive map: http://www.cobbgis.org:81/arcgislb1/rest/services/tax/cobbparcelmapwm/MapServer/0

**Comment by nvkelso** [comment]
Good find!

**Comment by nvkelso** [comment]
Drr, this was completed in https://github.com/openaddresses/openaddresses/issues/2609.

**Comment by justinelliotmeyers** [comment]
Thanks amigo!

**Comment by b-may** [comment]
According to my phone call with the GIS manager and this web page [1] - they assert copyright over their data. Did anyone get permission to utilize the data? Having said that, I heard last week there was some big get together among Georgia appraisers on releasing GIS data more freely - but appraiser offices and county GIS are different agencies. According to the state statutes on public records each agency can choose how they deal with the option for copyright and charging high fees. Again this was an exception added for GIS data. I've reached out to the state GIS Officer Susan Miller (470) 599-0631 susan.miller@gio.ga.gov - for her guidance and opinion, but did not receive a response yet.

1 - https://cobbcounty.org/index.php?option=com_content&view=article&id=5813&Itemid=2157

**Comment by iandees** [comment]
@b-may it's my (and other smart people I've talked about this with) opinion that the data we are collecting are facts, which cannot be copyrighted in the US. If a municipality puts their data on the internet without any other restriction (like a click-wrapper or contract), then we are free to use it however we want. In this case, we're using the [data they openly publish on their Esri server](http://www.cobbgis.org:81/arcgislb1/rest/services/tax/cobbparcelsmap_wgs/MapServer).

Having said that, we've pulled back data when requested at least once before. I'm more interested in working with municipalities towards open data policies than in being hardline about my opinion above.
